tests/main/views/test_search.py

```diff
@@ -571,7 +571,6 @@ def test_filter_form_given_category_selection(self):
             'lot': 'cloud-software',
             'serviceCategories': 'content management system (cms)',
             'parentCategory': 'electronic document and records management (edrm)',
-            'doc_type': 'services',
```

Comment:
😿 that we don't have the equivalent test for DOS (that's an ongoing sadface of mine)